app/models.py

```diff
@@ -1614,6 +1614,99 @@ def serialize(self):
         }
 
 
+class DirectAwardProject(db.Model):
+    __tablename__ = 'direct_award_projects'
+
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(255), nullable=False)
+    created_at = db.Column(db.DateTime, default=datetime.utcnow, nullable=False)
+    locked_at = db.Column(db.DateTime, nullable=True)  # When the project's active search/es are final and cannot change
+    active = db.Column(db.Boolean, default=True, nullable=False)
+
+    users = db.relationship(User, secondary='direct_award_project_users', order_by=lambda: DirectAwardProjectUser.id)
+
+    def serialize(self):
+        return {
+            "id": self.id,
+            "name": self.name,
+            "created_at": self.created_at.strftime(DATETIME_FORMAT),
+            "locked_at": self.locked_at.strftime(DATETIME_FORMAT) if self.locked_at is not None else None,
+            "active": self.active
+        }
+
+    @validates('name')
+    def _assert_active_project(self, key, value):
+        if self.locked_at:
+            raise ValidationError('Cannot change project name after locking it ({})'.format(self.id))
+
+        return value
+
+
+class DirectAwardProjectUser(db.Model):
+    __tablename__ = 'direct_award_project_users'
+
+    id = db.Column(db.Integer, primary_key=True)
+    project_id = db.Column(db.Integer, db.ForeignKey('direct_award_projects.id'), index=True, nullable=False)
+    user_id = db.Column(db.Integer, db.ForeignKey('users.id'), index=True, nullable=False)
+
+
+class DirectAwardSearch(db.Model):
+    __tablename__ = 'direct_award_searches'
+
+    id = db.Column(db.Integer, primary_key=True)
+    created_by = db.Column(db.Integer, db.ForeignKey('users.id'), nullable=False)
+    project_id = db.Column(db.Integer, db.ForeignKey('direct_award_projects.id'), nullable=False, index=True)
+    created_at = db.Column(db.DateTime, default=datetime.utcnow, nullable=False)  # When this search was first saved.
+    searched_at = db.Column(db.DateTime, nullable=True)  # When this search was run for the project services download.
+    search_url = db.Column(db.Text, nullable=False)  # Search API URL to hit when retrieving services.
```

Comment:
Yes, we considered this (and I was a proponent of the idea) - but given the search URL is the more canonical representation of what the user has done and the result set it describes, we opted to store that instead.